packages/template-retail-react-app/app/components/responsive/index.jsx

```diff
@@ -11,19 +11,27 @@ import {Box} from '@salesforce/retail-react-app/app/components/shared/ui'
 /**
  * Render the children in the DOM but visually hide them on desktop
  * @param children - isomorphic components used within a responsive design
+ * @param display - custom display property for the base and lg breakpoints
  */
-export const HideOnDesktop = ({children}) => (
-    <Box display={{base: 'block', lg: 'none'}}>{children}</Box>
+export const HideOnDesktop = ({children, display = {base: 'block', lg: 'none'}}) => (
```

Comment:
If we want to make it more general, we can have the user override any props. Something like this:

```jsx
export const HideOnDesktop = ({children, ...rest}) => (
    <Box display={{base: 'block', lg: 'none'}} {...rest}>
        {children}
    </Box>
)
```